index.bs

```diff
@@ -690,6 +706,8 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
 
     <li><i><dfn>Exposure Compensation</dfn></i> is a numeric camera setting that adjusts the exposure level from the current value used by the implementation.  This value can be used to bias the exposure level enabled by auto-exposure, and usually is a symmetric range around 0 EV (the no-compensation value).</li>
```

Comment:
This is only applicable if `exposureMode` is `continuous` or `single-shot`, right?